src/noc/noc.module.ts

```diff
@@ -37,7 +38,8 @@ import { effects, reducers } from './store';
       FormsModule,
       ReactiveFormsModule,
       UtilsModule,
-      ExuiCommonLibModule
+      ExuiCommonLibModule,
+      RpxTranslationModule.forChild(),
```

Comment:
not necessary; on the base branch the SharedModule already imports and exports RpxTranslationModule

Reply:
i tried without adding but it doesn't recognize the rpxTranslate so I have to add this.

Reply:
should work without.
If you have the SharedModule imported, you don't need to import RpxTranslationModule as you already import it (SharedModule exports RpxTranslationModule)